core/src/main/kotlin/org/evomaster/core/output/service/HttpWsTestCaseWriter.kt

```diff
@@ -220,6 +240,21 @@ abstract class HttpWsTestCaseWriter : ApiTestCaseWriter() {
         } else {
             addActionLines(call, lines, res, baseUrlOfSut)
         }
+
+        // reset all used external service action
+        if (exActions.isNotEmpty()) {
+            if (!format.isJavaOrKotlin()) {
+                log.warn("NOT support for other format ($format) except JavaOrKotlin")
```

Comment:
this log without context would be confusing... add something like `In mocking of external services, we do NOT ...`